app/javascript/stylesheets/login.css

```diff
@@ -0,0 +1,82 @@
+body {
+  background-color: #ffffff;
+}
+
+.login_content {
+  padding: 24px;
+  position: absolute;
+  display: block;
+  left: 50%;
+  transform: translate(-50%, 0%);
+  top: 23rem;
+  width: 100%;
+  text-align: center;
+}
+
+.scene-1-login {
+  position: absolute;
+  display: flex;
+  top: 8rem;
+  left: 50%;
+  transform: translate(-50%, 0%);
+  z-index: -1;
+}
+
+h1 {
+  font-weight: 600;
+  font-size: 21px;
+  display: flex;
+  text-align: center;
+  color: #252636;
+  justify-content: center;
+}
+
+form {
+  position: absolute;
+  left: 50%;
+  transform: translate(-50%, 0);
+  padding-top: 50px;
+  height: auto;
+  overflow: hidden;
+}
+
+form input {
+  float: left;
+  clear: both;
+  margin: 5px 0;
+  padding: 10px 170px 10px 20px;
+  outline: none;
+  border: 1px solid #252636;
+  border-radius: 5px;
+}
+
+input[type="text"]::-webkit-input-placeholder,
```

Comment:
**suggestion (blocking)**: most browsers already support the default `::placeholder` syntax and that is the preferred way to apply styles to the placeholder